src/services/bundleAnalysis/useBundleAssets.spec.tsx

```diff
@@ -345,4 +405,108 @@ describe('useBundleAssets', () => {
       )
     })
   })
+
+  describe('passing enabled to opts object', () => {
+    describe('enabled is true', () => {
+      describe('branch is not passed', () => {
+        it('does not run the query', async () => {
+          const { madeRequest } = setup({})
+          renderHook(
+            () =>
+              useBundleAssets({
+                provider: 'gh',
+                owner: 'codecov',
+                repo: 'codecov',
+                bundle: 'test-bundle',
+                opts: {
+                  enabled: true,
+                },
+              }),
+            { wrapper }
+          )
+
+          await waitFor(() => queryClient.isFetching())
+          await waitFor(() => !queryClient.isFetching())
+
+          await waitFor(() => expect(madeRequest).toHaveBeenCalled())
```

Comment:
Name of test implies query is not run, but the content of the test appears to be checking that the query did run. What am I missing here?

Reply:
Reading the below tests, I think this is just a test name issue!

Reply:
Actually I'm again not sure. I'll let you answer lol